packages/components/src/overlay/src/Underlay.css

```diff
@@ -5,6 +5,6 @@
     width: 100%;
     height: 100%;
     isolation: isolate;
-    background-color: var(--o-ui-bg-alias-underlay);
+    background-color: rgba(60, 60, 60, 0.6);
```

Comment:
Looks like they are using a variable in Figma, but I could actually find it in hopper styles of figma. Oh well.. maybe eventually this can be replaced

Reply:
Indeed, this variable was not exposed.